PythonRegression/tests/features/steps/api_test_steps.py

```diff
@@ -97,7 +131,7 @@ def compare_response(step):
             assert str(responseKeys[response_key_val]) == str(keys[response_key_val]['keys']), "There was an error with the response" 
     
     elif apiCall == 'getNeighbors' or apiCall == 'getTips':
-        responseList = responses[apiCall][nodeId] 
+        responseList = world.responses[apiCall][nodeId]
```

Comment:
IIUC the only difference between the 2 ifs is that in one case you return a response list, and in the other case there is a single response.

If you treat the single response as a list with size == 1 then you can remove the `if-elif` statement.